Sources/ComposableArchitecture/Documentation.docc/Articles/DependencyManagement.md

```diff
@@ -253,6 +253,25 @@ func testFetchUser() async {
 }
 ```
 
+Often times it is not necessary to create a whole new type to conform to `DependencyKey`. If the
+dependency you are registering is a type that you own, then you can conform it directly to the 
+protocol:
+
+```swift
+private enum APIClient: DependencyKey {
```

Comment:
Maybe a `struct` with a few endpoints would be more explicit than an empty `enum` here?

Reply:
Oops yes. Made this change as I was running out the door. Will flesh it out more tomorrow!